src/components/sdk/MeetingControls/AudioOutputControl.tsx

```diff
@@ -26,28 +25,40 @@ const AudioOutputControl: React.FC<Props> = ({
   const selectAudioOutput = useSelectAudioOutputDevice();
   const { devices, selectedDevice } = useAudioOutputs();
   const { isAudioOn, toggleAudio } = useLocalAudioOutput();
+  const [dropdownOptions, setDropdownOptions] = useState<PopOverItemProps[]>(
+    []
+  );
 
-  const handleClick = async (deviceId: string): Promise<void> => {
-    try {
-      if (new DefaultBrowserBehavior().supportsSetSinkId()) {
-        await selectAudioOutput(deviceId);
-      } else {
+  useEffect(() => {
```

Comment:
Any reason for this change? I think commented somewhere on AudioInputControl as well.